x-pack/plugins/maps/public/routes/map_page/top_nav_config.tsx

```diff
@@ -240,6 +240,14 @@ export function getTopNavConfig({
             <SavedObjectSaveModalDashboard
               {...saveModalProps}
               canSaveByReference={true} // we know here that we have save capabilities.
+              mustCopyOnSaveMessage={
+                savedMap.isManaged()
+                  ? i18n.translate('xpack.maps.topNav.mustCopyOnSaveMessage', {
+                      defaultMessage:
+                        'This map is managed by Elastic. Changes here must be saved to a new map.',
```

Comment:
Nit - how about removing "here" so that "Changes here must be saved to a new map." becomes "Changes must be saved to a new map."